examples/drivers/windows/otLwf/iocontrol.c

```diff
@@ -5270,8 +5270,14 @@ otLwfIoCtl_otJoinerStart(
     if (InBufferLength >= sizeof(otCommissionConfig))
     {
         otCommissionConfig *aConfig = (otCommissionConfig*)InBuffer;
-        status = ThreadErrorToNtstatus(otJoinerStart(
-            pFilter->otCtx, (const char*)aConfig->PSKd, (const char*)aConfig->ProvisioningUrl));
+        status = ThreadErrorToNtstatus(
+            otJoinerStart(
+                pFilter->otCtx,
+                (const char*)aConfig->PSKd,
+                (const char*)aConfig->ProvisioningUrl,
+                NULL,
+                NULL)
+            );
```

Comment:
Could you just put a TODO comment at all these places that need to be finished up?